wadors:  accept header (example) and properties type from imagePixelModule

lscoder opened the pull request.

lscoder commented on src/imageLoader/wadors/loadImage.js:
```diff
@@ -23,7 +23,7 @@
 
     // TODO: load bulk data items that we might need
 
-    var mediaType = 'multipart/related; type="application/octet-stream"'; // 'image/dicom+jp2';
```
The quotation marks were added by https://github.com/chafey/cornerstoneWADOImageLoader/commit/6a792c01dc716d44c80980edb2d2f257c04cae6b commit but it's breaking Orthanc. I tried with double and single quotes but they didn't work.

swederik replied:
So is this a bug in Orthanc, then? I believe the original PR (https://github.com/chafey/cornerstoneWADOImageLoader/issues/54) said that this followed a correction in the DICOM Standard. 

I guess we should report it there instead of removing the quotes here.

lscoder commented:
I added the Content-Type quotes back because after checking Orthanc DICOMWeb code I saw a commit from Sébastien Jodogne ([e351748](https://bitbucket.org/sjodogne/orthanc-dicomweb/commits/e351748bdbd45f773e8b80d941bbe530a241421b)) that fix this issue. Then I downloaded the latest version, built a new DICOMWeb plugin, updated my Orthanc and everything worked fine.
